frontend/build-webui.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+. ./setup-nodejs-env.sh
```

Comment:
typo?

Reply:
I didn't touch this line, it looks like it's sourcing the setup-nodejs-env.sh file instead of executing it. I don't think it's a typo.